providers/loxo/README.md

```diff
@@ -0,0 +1,66 @@
+| company_global_statuses                   | company_global_statuses                  | read       |
+| company_types                             | company_types                            | read       |
+| compensation_types                        | compensation_types                       | read       |
+| countries                                 | countries                                | read       |
+| currencies                                | currencies                               | read       |
+| deal_workflows                            | deal_workflows                           | read       |
+| deals                                     | deals                                    | read       |
+| disability_statuses                       | disability_statuses                      | read       |
+| diversity_types                           | diversity_types                          | read       |
+| dynamic_fields                            | dynamic_fields                           | read       |
+| education_types                           | education_types                          | read       |
+| email_tracking                            | email_tracking                           | read       |
+| email_types                               | email_types                              | read       |
+| equity_types                              | equity_types                             | read       |
+| ethnicities                               | ethnicities                              | read       |
+| fee_types                                 | fee_types                                | read       |
+| form_templates                            | form_templates                           | read       |
+| forms                                     | forms                                    | read       |
+| genders                                   | genders                                  | read       |
+| job_categories                            | job_categories                           | read       |
+| job_contact_types                         | job_contact_types                        | read       |
+| job_owner_types                           | job_owner_types                          | read       |
+| job_statuses                              | job_statuses                             | read       |
+| job_types                                 | job_types                                | read       |
+| jobs                                      | jobs                                     | read       |
+| people                                    | people                                   | read       |
+| people/emails                             | people/emails                            | read       |
+| person_phones                             | person_phones                            | read       |
+| people/update_by_email                    | people/update_by_email                   | read       |
```

Comment:
Either we should merge this as an update with the people object, or not support this object, because this is an action/verb, not a resource